sqlglot/dialects/prql.py

```diff
@@ -78,6 +79,22 @@ def _parse_selection(self, query: exp.Query, append: bool = True) -> exp.Query:
 
             return query.select(*selects, append=append, copy=False)
 
+        def _parse_take(self, query: exp.Query) -> exp.Query:
+            if self._match(TokenType.NUMBER):
+                first = self._prev.text  # TAKE 4..10 -> self._prev.text = "4."
+            else:
+                self.raise_error("Expecting a number")
+
+            if self._match_pair(TokenType.DOT, TokenType.NUMBER):  # TAKE <start>..<end>
+                first = int(first[:-1])
+                second = int(self._prev.text)
+                if second >= first:  # valid range
+                    return query.limit(second - first + 1).offset(first - 1)
+                else:
+                    return query.limit(0)
```

Comment:
Why is this doing `limit(0)`?

Reply:
From the [PRQL Playground](https://prql-lang.org/playground/), the `from x take 1..0` seems to be a valid query and produce `select * from x limit 0`, so I want to handle that here.